### PR TITLE
[Wont Merge?] Corrected ssb-broadcast-service

### PR DESCRIPTION
**Pull request by darkdrgn2k**
@ASoTNetworks submitted patch over chat 

> Think I got it working and tcpdump shows its sending over ygg and cjdn

### CODE REVIEW

**Comment by darkdrgn2k** [comment]
Smaller patch already applied that fixes issue.

This code MAY be easier to read? 

Thoughts?

**Comment by makeworld-the-better-one** [comment]
I don't feel there's a large advantage in writing it this way, the original was fine.

File: scripts/ssb/ssb-broadcast-service.sh [hunk 1]
```diff
@@ -12,10 +12,14 @@ while true; do
     fi
 
     # Manual cjdns peer unicast
-    if [ "$(which cjdns)" ]; then
+    if [ "$(which cjdroute)" ]; then
         mycjdnsip=$(grep -m 1 '"ipv6"' /etc/cjdroute.conf | awk '{ print $2 }' | sed 's/[",]//g')
         # shellcheck disable=SC2102,SC2046
-        read -ar peers <<< $(sudo nodejs /opt/cjdns/tools/peerStats 2>/dev/null | awk '{ if ($3 == "ESTABLISHED") print $2 }' | awk -F. '{print $6".k"}' | xargs)
+        #read -ar peers <<< $(sudo nodejs /opt/cjdns/tools/peerStats 2>/dev/null | awk '{ if ($3 == "ESTABLISHED") print $2 }' | awk -F. '{print $6".k"}' | xargs)
```
Comment (benhylau) [inline scripts/ssb/ssb-broadcast-service.sh]:
```suggestion
```

File: scripts/ssb/ssb-broadcast-service.sh [hunk 2]
```diff
@@ -26,7 +30,11 @@ while true; do
     # Add yggdrasil direct peers    
     if [ "$(which yggdrasil)" ]; then       
         myyggip=$(yggdrasilctl getself | grep address | awk '{print $3}')        
-        read -ar peers  <<< "$(sudo yggdrasilctl getPeers | grep -v "(self)" | awk '{print $1}' | grep -v bytes_recvd | xargs)"
+        #read -ar peers  <<< "$(sudo yggdrasilctl getPeers | grep -v "(self)" | awk '{print $1}' | grep -v bytes_recvd | xargs)"
```
Comment (benhylau) [inline scripts/ssb/ssb-broadcast-service.sh]:
```suggestion
```

**Comment by benhylau** [comment]
I am good with closing or merging, but if we merge we should remove that comment (see my suggestions).

**Pull request closed without merging**